CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- Fix mutation logic for `giantswarm.io/cluster` label for org-namespaced App CRs.
```

Comment:
```suggestion
### Changed

- Adapt mutation logic for `giantswarm.io/cluster` label to support App CRs in the org namespace.
```

Nit but I'd say supporting the org namespace is new functionality rather than a fix.

Reply:
Makes sense, thanks!